### PR TITLE
Hotel models

### PR DESCRIPTION
**Pull request by leny62**
**What does this PR do?**
Creates hotel models

**Description of Task to be completed?**
-creates hotel models, that a user will be able to perform CRUD operations with hotels.
- a user requests a list of hotels.
- a user gets different details of hotels
- one is able to get, delete and update hotels.
**How should this be manually tested?**
1.Clone the repository/ Pull this branch
2.In your home directory move into the cloned folder by cd hotel-models
3.Then run npm i to install all dependencies.
4. Modify the connection string to database by creating a .env file, Refer to data in .env.example file.
5.Run npm start to start the server.
6. test the routes via postman
**Any background context you want to provide?**
- I will create tests after receiving PR reviews.
- No authentication required at the moment.
**What are the relevant Pivotal Tracker stories?**
[#175930848](https://www.pivotaltracker.com/story/show/175930848)
Screenshots (if appropriate)
Screen Shot 2020-12-16 at 15 54 35

### CODE REVIEW

**Review by furebo — COMMENTED** [review]
Please write the tests for endpoints so that they run correctly on travis

File: src/app.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
 import express from 'express';
+
 import dotenv from 'dotenv';
 
+import hotelRoutes from './routes/hotelRoute';
+
+// const createHotel = require('./routes/hotelRoute');
+// const getHotels = require('./routes/hotelRoute');
+// const deleteHotel = require('./routes/hotelRoute');
+// const getHotel = require('./routes/hotelRoute');
```
Comment (Anguandia) [inline src/app.js]: Please remove commented out code

**Review by Anguandia — COMMENTED** [review]
Please complete the tests

File: src/database/Models/hotel.js [hunk 2]
```diff
@@ -13,12 +13,19 @@ module.exports = (sequelize, DataTypes) => {
...
+  hotel.init({
+    hotelName: DataTypes.STRING,
+    priceRange: DataTypes.STRING,
+    location: DataTypes.STRING,
+    ranking: DataTypes.STRING,
+    parking: DataTypes.STRING,
+    wifi: DataTypes.STRING,
+    swimmingPool: DataTypes.STRING,
+    breakfast: DataTypes.STRING,
+    rooms: DataTypes.ARRAY(DataTypes.STRING)
```
Comment (SimonImanigirimpuhwe) [inline src/database/Models/hotel.js]: I think you need also to add hotel avatar since we will need image of the hotel to be displayed on the front-end.

**Comment by Anguandia** [comment]
Please check your travis

**Review by jemmyarina — COMMENTED** [review]
Your tests on travis are failling. Re-work on it

File: src/database/Models/hotel.js (hunk 2 above)
Comment (leny62) [reply src/database/Models/hotel.js]: Resolved!

**Review by furebo — CHANGES_REQUESTED** [review]
Please writes the tests correctly so that the travis build will pass

File: src/controllers/hotelController.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import model from '../database/Models';
+
+require('dotenv').config();
```
Comment (Anguandia) [inline src/controllers/hotelController.js:3]: please convert all ES5 code within /src to ES6

**Review by Anguandia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**